src/jsutils/symbolSupport.js

```diff
@@ -0,0 +1,80 @@
+/**
+ * Copyright (c) 2015-present, Facebook, Inc.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @flow strict
+ */
+
+/**
+ * A function that can either simply determine if `Symbol`s are allowed as
+ * well as, optionally, determine whether or not a property of symbol
+ * exists.
+ *
+ * If the name of a specific `Symbol` property is supplied, the resulting
+ * value will only be true if property is mapped to an instance of `Symbol`
+ * such as `.toStringTag`, `.iterator`, `.species`, `.isConcatSpreadable`,
+ * etc...
+ *
+ * @method hasSymbolSupport
+ *
+ * @param {string}  specificSymbol an optional name of a property on the
+ * `Symbol` class itself that statically refers to a predefined `Symbol`. If
+ * properly specified and the value is set, then true will be returned.
+ * @return {bool} true if `Symbol` is both defined and a function. Optionally
+ * true if the `Symbol` class is true, a predefined symbol name such as
+ * `toStringTag` is set on the `Symbol` class and it maps to an instance of
+ * `Symbol`. False in all other cases
+ */
+export function hasSymbolSupport(specificSymbol?: string): boolean {
```

Comment:
Since `hasSymbolSupport` was used only by test that would be removed you don't need to export this function anymore.
This allows making `applyToStringTag` default export similar to all other `jsutils/*` files.
\+ it would make sense to rename this file into `applyToStringTag.js`